src/components/Key/Key.css

```diff
@@ -1,11 +1,11 @@
 .key {
-  background-color: rgba(255, 255, 255, 0.671);
+  background-color: rgba(255, 255, 255, 0.849);
   height: 20rem;
   flex-grow: 1;
   min-width: 5em;
   margin: 0 .1em;
   border-radius: 5px;
-  /* position: absolute; */
+  box-shadow: 0px 8px 4px 1px rgba(0, 0, 0, 0.445);
```

Comment:
I might want to roll back the shadow a little bit, they currently look really far off the keyboard, but they are a great addition that hand't even crossed my mind before!